Tests/App/security.php

```diff
@@ -9,10 +9,12 @@
  * file that was distributed with this source code.
  */
 
+use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\Security\Core\Security;
 use Symfony\Component\Security\Http\Controller\UserValueResolver;
 
-if (method_exists(Security::class, 'getUser') && !class_exists(UserValueResolver::class)) {
+if (isset($container) && method_exists(Security::class, 'getUser') && !class_exists(UserValueResolver::class)) {
+    /* @var ContainerBuilder $container */
```

Comment:
Or instead of above `isset()` change declare this before the `if`.

Reply:
Not sure I follow

Reply:
The error was

```
 ------ ------------------------------------------- 
  Line   Tests/App/security.php                     
 ------ ------------------------------------------- 
  18     Variable $container might not be defined.  
 ------ ------------------------------------------- 
```

Reply:
Okey, I see this thing should be removed et all & put into the Kernel, will do PR.